DataCollection/getting_started.ipynb

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
This notebook will need to contain the complete pipeline to show how these functions work together